mne/viz/tests/test_raw.py

```diff
@@ -273,11 +260,11 @@ def test_plot_raw_selection(raw, new_mpl):
     assert len(fig.mne.traces) == len(sel_dict['Left-temporal'])  # unchanged
     assert buttons.value_selected == before_state                 # unchanged
     # test marking bad channel in selection mode → should make sensor red
-    assert sel_fig.lasso.ec[:, 0].sum() == 0  # R of RGBA zero for all chans
-    _click_ch_name_helper(fig, ch_index=1, button=1)  # mark bad
-    assert sel_fig.lasso.ec[:, 0].sum() == 1          # one channel red
-    _click_ch_name_helper(fig, ch_index=1, button=1)  # mark good
-    assert sel_fig.lasso.ec[:, 0].sum() == 0          # all channels black
+    assert sel_fig.lasso.ec[:, 0].sum() == 0   # R of RGBA zero for all chans
+    _click_ch_name(fig, ch_index=1, button=1)  # mark bad
+    assert sel_fig.lasso.ec[:, 0].sum() == 1   # one channel red
+    _click_ch_name(fig, ch_index=1, button=1)  # mark good
+    assert sel_fig.lasso.ec[:, 0].sum() == 0   # all channels black
     # test lasso
     sel_fig._set_custom_selection()  # lasso empty → should do nothing
     sensor_ax = sel_fig.mne.sensor_ax
```

Comment:
Okay now it looks like you can fix the test below:

https://travis-ci.org/github/mne-tools/mne-python/jobs/742563175#L6462-L6469

```
mne/viz/tests/test_raw.py:280: in test_plot_raw_selection
    assert got == want
E   AssertionError: assert ['MEG 0121', 'MEG 0122', 'MEG 0123'] == ['MEG 0111', 'MEG 0112', 'MEG 0131', 'MEG 0132', 'MEG 0133']
E     At index 0 diff: 'MEG 0121' != 'MEG 0111'
E     Right contains 2 more items, first extra item: 'MEG 0132'
E     Full diff:
E     - ['MEG 0111', 'MEG 0112', 'MEG 0131', 'MEG 0132', 'MEG 0133']
E     + ['MEG 0121', 'MEG 0122', 'MEG 0123']
```